clarifai/client/app.py

```diff
@@ -288,6 +288,42 @@ def list_concepts(self, page_no: int = None,
       concept_info['id'] = concept_info.pop('concept_id')
       yield Concept(**concept_info)
 
+  def list_concept_relations(self,
+                             concept_id: str = None,
+                             predicate: str = None,
```

Comment:
Can we set predicate to optional, passing `None` to the request data is not the better way to do so. We can pass empty string in predicate which will work but not when directly given as `None`, 
When empty string is passed in predicate it will list all the relations with respect to that concept.